mobile/src/components/courses/courses.tsx

```diff
@@ -50,13 +58,32 @@ const Courses: FC = (): ReactElement => {
     dataStatus === DataStatus.PENDING ||
     categoryDataStatus === DataStatus.PENDING;
 
-  const filter = useRef<CourseFilteringDto>({
+  const filter = useRef<CourseFilteringWithPaginationDto>({
     title: '',
     categoryKey: '',
+    page: PaginationDefaultValue.DEFAULT_PAGE,
+    count: PaginationDefaultValue.DEFAULT_COUNT_BY_20,
   });
 
   const handleCoursesLoad = (): void => {
-    dispatch(coursesActions.getCourses(filter.current));
+    dispatch(
+      coursesActions.getCourses({
+        title: filter.current.title ?? '',
+        categoryKey: filter.current.categoryKey ?? '',
+        page: filter.current.page,
+        count: filter.current.count,
+      }),
+    );
+  };
+
+  const handleLoadMore = (): void => {
+    const isExistMoreCourses = totalCoursesNumber > filter.current.count;
+
+    if (isExistMoreCourses) {
+      filter.current.count =
+        filter.current.count + PaginationDefaultValue.DEFAULT_COUNT_BY_20;
```

Comment:
Won't it work the same way?
```suggestion
      filter.current.count += PaginationDefaultValue.DEFAULT_COUNT_BY_20;
```